lib/commands/setup.js

```diff
@@ -669,6 +669,10 @@ SetupScreens.prototype.check = function check(callback) {
 				var cwd = process.mainModule && process.mainModule.filename,
 					root = path.resolve('/');
 
+				if (path.basename(cwd) === 'appc') {
+					cwd = require.resolve('titanium');
```

Comment:
Why can't you just do `cwd = path.resolve(__dirname, '..', '..');`?

Reply:
That makes more sense, and removes the need to even have this check. Updating